CGATPipelines/pipeline_bamstats/pipeline.ini

```diff
@@ -63,7 +57,7 @@ name=csvdb
 #database location 
 database=/ifs/projects/adam/pipeline_publication/csvdb
 
-# location that pipeline_bamstats.py was ran
+# location that pipeline_genesets.py was ran in
 dir=/ifs/projects/adam/pipeline_publication
```

Comment:
I think we should store this information in `/ifs/mirror` or similar but not on a project specific folder

Reply:
Ok agreed, ill push new changes